gbpservice/nfp/core/controller.py

```diff
@@ -374,7 +377,7 @@ def post_event_graph(self, event, graph_nodes):
             LOG.debug(message)
             self._manager.process_events([event])
 
-    def post_event(self, event):
+    def post_event(self, event, target=None):
```

Comment:
What is target here? Does target indicate who should be handling it? There should be a separation between who is posting and who is handling..